python/GangaLHCb/Lib/RTHandlers/RTHUtils.py

```diff
@@ -57,12 +57,6 @@ def lhcbdiracAPI_script_template():
 def is_gaudi_child(app):
     if isType(app, Gaudi):
         return True
-
-    if isType(app, TaskApplication):
-        from GangaLHCb.Lib.Applications import GaudiPythonTask, BenderTask
-        if not isType(app, GaudiPythonTask) and not isType(app, BenderTask):
```

Comment:
@milliams Can we bump this to 6.1.19 then so that this release works for LHCb people with Tasks I'd like to get as many people off 6.0.44 for LHCb asap.